x/merkledb/benchmarks/benchmark.go

```diff
@@ -0,0 +1,404 @@
+			addDuration := time.Since(startInsertBatchTime)
+			insertRate.Set(float64(databaseCreationBatchSize) * float64(time.Second) / float64(addDuration))
+			insertsCounter.Add(databaseCreationBatchSize)
+
+			batchWriteStartTime := time.Now()
+			err = currentBatch.Write()
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "unable to write value in merkleDB database : %v\n", err)
+				return err
+			}
+			currentBatch.Reset()
+			batchWriteDuration := time.Since(batchWriteStartTime)
+			batchWriteRate.Set(float64(time.Second) / float64(batchWriteDuration))
+			batchCounter.Inc()
+
+			startInsertBatchTime = time.Now()
+		}
+	}
+	if (*databaseEntries)%databaseCreationBatchSize != 0 {
+		err = currentBatch.Write()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "unable to write value in merkleDB database : %v\n", err)
+			return err
+		}
+	}
+	close(ticksCh)
+
+	fmt.Print(" done!\n")
+	rootID, err := mdb.GetMerkleRoot(context.Background())
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to get root ID : %v\n", err)
+		return err
+	}
+
+	fmt.Printf("Generated and inserted %d batches of size %d in %v\n",
+		(*databaseEntries)/databaseCreationBatchSize, databaseCreationBatchSize, time.Since(startInsertTime))
+	err = mdb.Close()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to close merkleDB database : %v\n", err)
+		return err
+	}
+	err = levelDB.Close()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to close levelDB database : %v\n", err)
+		return err
+	}
+	err = os.Rename(getGoldenStagingDatabaseDirectory(), getGoldenDatabaseDirectory())
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to rename golden staging directory : %v\n", err)
+		return err
+	}
+	fmt.Printf("Completed initialization with hash of %v\n", rootID.Hex())
+	return nil
+}
+
+func resetRunningDatabaseDirectory() error {
+	runningDir := getRunningDatabaseDirectory()
+	if _, err := os.Stat(runningDir); err == nil {
+		err := os.RemoveAll(runningDir)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "unable to remove running directory : %v\n", err)
+			return err
+		}
+	}
+	err := os.Mkdir(runningDir, 0o777)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to create running directory : %v\n", err)
+		return err
+	}
+	err = CopyDirectory(getGoldenDatabaseDirectory(), runningDir)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to duplicate golden directory : %v\n", err)
+		return err
+	}
+	return nil
+}
+
+func runBenchmark() error {
+	levelDB, err := leveldb.New(
+		getRunningDatabaseDirectory(),
+		getLevelDBConfig(),
+		logging.NoLog{},
+		promRegistry,
+	)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to create levelDB database : %v\n", err)
+		return err
+	}
+
+	mdb, err := merkledb.New(context.Background(), levelDB, getMerkleDBConfig(promRegistry))
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to create merkleDB database : %v\n", err)
+		return err
+	}
+	defer func() {
+		mdb.Close()
+		levelDB.Close()
+	}()
+
+	low := uint64(0)
```

Comment:
`low` never changes, and should be increased by 2.5k each pass

Reply:
good catch; fixed.